docs/shared_parsers_catalog/neutron_server_log.rst

```diff
@@ -0,0 +1,9 @@
+Neutron Server Log
+==================
+
+NeutronServerLog - ``/var/log/neutron/server.log``
+--------------------------------------------------
+.. autoclass:: insights.parsers.neutron_server_log.NeutronServerLog
+   :members:
+   :show-inheritance:
+
```

Comment:
OK, following from the docs discussion elsewhere let's get this sorted out before we start trying things out randomly :-)  I'll go back to the previous docs system.